<!-- -->
<launch>
  <arg name='host' default='localhost'/>
  <arg name='port' default='2000'/>
  <arg name='timeout' default='10'/>
  <arg name="actors_definition_file"/>
  <arg name="role_name" default="ego_vehicle"/>
  <!-- use comma separated format "x,y,z,roll,pitch,yaw" -->
  <arg name="spawn_point" default="392.5,-87.0,0.3,0.0,0.0,90.0"/>

  <node pkg="carla_spawn_actors" type="carla_spawn_actors.py" name="carla_spawn_actors_$(arg role_name)" output="screen">
    <param name="carla/host" value="$(arg host)" />
    <param name="carla/port" value="$(arg port)" />
    <param name="carla/timeout" value="$(arg timeout)" />
    <param name="actors_definition_file" value="$(arg actors_definition_file)" />
    <param name="role_name" value="$(arg role_name)" />
    <param name="spawn_point" value="$(arg spawn_point)" />
  </node>
</launch>
